android_kernel_tools.rc

```diff
@@ -502,9 +508,23 @@ function bootzip()
   signzip "${TmpDir}/${OutFile}.unsigned.zip" "${TmpDir}/${OutFile}";
   cp "${TmpDir}/${OutFile}" "${TargetFile}";
 
+  if [[ ! -z ${PushToDevice} ]]; then
```

Comment:
"" around the variable that can be empty (cleaner) and a "# Push zip to the device" above